src/collectors/diskspace/diskspace.py

```diff
@@ -156,8 +161,12 @@ def get_file_systems(self):
                                        mount_point)
                         continue
 
-                    if (major, minor) in result:
+                    if (major, minor) in result \
+                            and mount_point not in self.no_major_minor:
```

Comment:
As a matter of fact I *did* mean to use fs_type. This patch came across terribly!

Thanks.